packages/core/src/runtime.ts

```diff
@@ -336,6 +336,29 @@ export function workflowEntrypoint(workflowCode: string) {
             // Load all events into memory before running
             const events = await getAllWorkflowRunEvents(workflowRun.runId);
 
+            // Check for any elapsed timers and create timer_completed events
```

Comment:
I feel like we should include the `correlationId` in the queue payload when re-invoking the workflow. That way, you'd only check for the specific one that is supposed to have elapsed here. Would probably be useful for other kinds of metrics/debugging as well.